extensions/amp-ima-video/0.1/validator-amp-ima-video.protoascii

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: { #amp-ima-video
+  tag_name: "script"
+  spec_name: "amp-ima-video extension .js script"
+  mandatory_parent: "head"
+  attrs: {
+    name: "async"
+    mandatory: true
+    value: ""
+  }
+  attrs: {
+    name: "custom-element"
+    mandatory: true
+    value: "amp-ima-video"
+    dispatch_key: true
+  }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_regex: "https://cdn\\.ampproject\\.org/v0/amp-ima-video-(latest|0\\.1).js"
+  }
+  attrs: {
+    name: "type"
+    value: "text/javascript"
+  }
+  cdata: {
+    blacklisted_cdata_regex: {
+      regex: "."
+      error_message: "contents"
+    }
+  }
+  spec_url: "https://www.ampproject.org/docs/reference/extended/amp-ima-video.html"
+}
+tags: {  # <amp-ima-video>
```

Comment:
A couple of changes are needed:
- specify an `html_format`
- the `tag_name` and `disallowed_ancestor` values need to be capitalized
- `disallowed_ancestor: head` isn't necessary because any body-only tags in `head` causes the `body` to automatically begin.
- we moved from `also_requires` to a `satisfies` and `requires` model

That changes things to:
```
  html_format: AMP
  tag_name: "AMP-IMA-VIDEO"
  disallowed_ancestor: "AMP-SIDEBAR"
  satisfies: "amp-ima-video"
  requires: "amp-ima-video extension .js script"
```

Also note the restriction to data-src URL being https that is mentioned below.